packages/core/src/generators/interface.ts

```diff
@@ -53,6 +53,7 @@ export const generateInterface = ({
   } else {
     model += `export type ${name} = ${scalar.value};\n`;
   }
+  model += `export function create${name}(): ${name} ${scalar.factoryMethodValue}\n`;
```

Comment:
You will add this in the model file here. It's a bit weird no?

Reply:
Yeah... was thinking whether it should be in seperate file or not. I've decided to leave it with model. Anyone who has imported a model, can either instantiate it through "new", or right away call a factory method - which increases the chances that random dev will learn about it and use it.

People are lazy and do not read the code - especially the one that's generated.